src/Tests.Microsoft.Security.Utilities.Core/SecretMaskerTests.cs

```diff
@@ -143,23 +144,33 @@ private void ValidateSecurityModelsMasking(IEnumerable<RegexPattern> patterns, I
                         {
                             string secretValue = testExample;
                             string moniker = pattern.GetMatchMoniker(secretValue);
-                            string sha256Hash = RegexPattern.GenerateSha256Hash(secretValue);
 
                             // 1. All generated test patterns should be detected by the masker.
                             string redacted = secretMasker.MaskSecrets(secretValue);
                             bool result = redacted.Equals(secretValue);
                             result.Should().BeFalse(because: $"'{secretValue}' for '{moniker}' should be redacted from scan text.");
+
+                            if (testExample.Contains(";"))
```

Comment:
// TODO: the generated examples don't distinguish the secret from surrounding context,
                            // for rules such as our connection string detecting logic. We need a future change to
                            // separate this data. For now, we skip all connection string patterns. This is a problem
                            // for masking only (and not detection) because we have no location details when masking.